<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1466</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 12, 36)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLineEdit" name="lineEdit_expression">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 7, 22);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>831</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="tok_buttons_grid">
     <item row="3" column="1">
      <widget class="QPushButton" name="atan_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>atan</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="log_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="mod_button_p">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>mod</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="three_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="x_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="six_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="four_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QPushButton" name="zero_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="leftp_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="mult_button_p">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="acos_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>acos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="tan_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="cos_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="sqrt_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="rightp_button_p">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="plus_button_p">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QPushButton" name="nine_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="e_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>e</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="div_button_p">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="sin_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="two_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="power_button_p">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="five_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="remove_button_o">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="ln_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="dot_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="4">
      <widget class="QPushButton" name="deposit_button_o">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(34, 17, 67);color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Deposit calculator</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="seven_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QPushButton" name="credit_button_o">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(34, 17, 67);color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Credit calculator</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="eight_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="minus_button_p">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="asin_button_f">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>asin</string>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QPushButton" name="eq_button_o">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="one_button_n">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(58, 38, 96);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>10</y>
      <width>186</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="graphError_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="miny_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>min y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_maxy">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 7, 22);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_miny">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 7, 22);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>-10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="maxy_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>max y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_maxx">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 7, 22);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_minx">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 7, 22);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>-10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="maxx_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>max x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="minx_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>min x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="build_button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(34, 17, 67);color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>build</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="minx_label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_xvalue">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 7, 22);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-100000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
